Tests/TestSuites/MapNodeTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+@testable import StencilSwiftKit
+
+class MapNodeTests: XCTestCase {
+  static let context = [
+    "items1": ["one", "two", "three"],
+    "items2": ["hello", "world", "everyone"]
+  ]
+  
+  func testBasic() {
+    let template = SwiftTemplate(templateString: Fixtures.string(for: "map-basic.stencil"), environment: stencilSwiftEnvironment())
+    let result = try! template.render(MapNodeTests.context)
+    
+    let expected = Fixtures.string(for: "map-basic.out")
+    XCTDiffStrings(result, expected)
```

Comment:
I'm wondering if testing Stencil Nodes using templates is a good thing.
When comes the time when we decide to move the MapNode to Stencil itself, Stencil won't have our fixtures as templates to test anyway.

The way Stencil tests its nodes is by invoking the Node constructor, then its render method, and test the result, without needing a template fixture for node unit tests. [See here](https://github.com/kylef/Stencil/blob/master/Tests/StencilTests/ForNodeSpec.swift#L14-L16). Maybe we should test at the same level too.